# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1398599124 -7200
#      Sun Apr 27 13:45:24 2014 +0200
# Branch gui-release
# Node ID 03edfcd943dbe4de8cb2b937331eb1b3cf8a801b
# Parent  86eca5d178a669afb7a3664270212804dee13d37
fix setting shortcuts for the view menu in the editor

* file_editor.cc (set_shortcuts): correct the keys of the related settings

* shortcut_manager.cc (do_init_data): new default for zoom normal shortcut

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -1571,19 +1571,19 @@ file_editor::set_shortcuts ()
   shortcut_manager::set_shortcut (_toggle_bookmark_action, "editor_edit:toggle_bookmark");
   shortcut_manager::set_shortcut (_next_bookmark_action, "editor_edit:next_bookmark");
   shortcut_manager::set_shortcut (_previous_bookmark_action, "editor_edit:previous_bookmark");
   shortcut_manager::set_shortcut (_remove_bookmark_action, "editor_edit:remove_bookmark");
   shortcut_manager::set_shortcut (_preferences_action, "editor_edit:preferences");
   shortcut_manager::set_shortcut (_styles_preferences_action, "editor_edit:styles_preferences");
 
   // View menu
-  shortcut_manager::set_shortcut (_zoom_in_action, "edit_edit:zoom_in");
-  shortcut_manager::set_shortcut (_zoom_out_action, "edit_edit:zoom_out");
-  shortcut_manager::set_shortcut (_zoom_normal_action, "edit_edit:zoom_normal");
+  shortcut_manager::set_shortcut (_zoom_in_action, "editor_view:zoom_in");
+  shortcut_manager::set_shortcut (_zoom_out_action, "editor_view:zoom_out");
+  shortcut_manager::set_shortcut (_zoom_normal_action, "editor_view:zoom_normal");
 
   // Debug menu
   shortcut_manager::set_shortcut (_toggle_breakpoint_action, "editor_debug:toggle_breakpoint");
   shortcut_manager::set_shortcut (_next_breakpoint_action, "editor_debug:next_breakpoint");
   shortcut_manager::set_shortcut (_previous_bookmark_action, "editor_debug:previous_breakpoint");
   shortcut_manager::set_shortcut (_remove_all_breakpoints_action, "editor_debug:remove_breakpoints");
 
   // Run menu
diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -147,17 +147,17 @@ shortcut_manager::do_init_data ()
   init (tr ("Remove All Bookmark"), "editor_edit:remove_bookmark",  QKeySequence () );
 
   init (tr ("Preferences"), "editor_edit:preferences",  QKeySequence () );
   init (tr ("Styles Preferences"), "editor_edit:styles_preferences",  QKeySequence () );
 
   // view
   init (tr ("Zoom In"), "editor_view:zoom_in",  QKeySequence::ZoomIn );
   init (tr ("Zoom Out"), "editor_view:zoom_out",  QKeySequence::ZoomOut );
-  init (tr ("Zoom Normal"), "editor_view:zoom_normal",  QKeySequence (Qt::ControlModifier + Qt::Key_Slash) );
+  init (tr ("Zoom Normal"), "editor_view:zoom_normal",  QKeySequence (Qt::ControlModifier + Qt::AltModifier + Qt::Key_0));
 
   // debug
   init (tr ("Toggle Breakpoint"), "editor_debug:toggle_breakpoint",  QKeySequence () );
   init (tr ("Next Breakpoint"), "editor_debug:next_breakpoint",  QKeySequence () );
   init (tr ("Previous Breakpoint"), "editor_debug:previous_breakpoint",  QKeySequence () );
   init (tr ("Remove All Breakpoints"), "editor_debug:remove_breakpoints",  QKeySequence () );
 
   // run
